### ISSUE TITLE
0.8.2 Windows binaries

### ISSUE THREAD

**Issue by magnunor** [hyperspy/hyperspy#724]
There are no Windows binaries on the https://github.com/hyperspy/hyperspy/releases/ page, or http://hyperspy.org/download.html#beta-releases

**Comment by magnunor** [comment]
After reading the release issue (https://github.com/hyperspy/hyperspy/issues/705), I'm guessing the intention is downloading 0.8.1, then upgrading it to 0.8.2 through PIP?

**Comment by francisco-dlp** [comment]
The windows binaries are actually missing and we need to fix this as it is confusing for the users.

**Comment by jrminter** [comment]
I needed to re-install recently and followed magnunor's suggestion. 

Then today I saw the blog post here:

http://stiglerdiet.com/blog/2015/Nov/24/my-python-environment-workflow-with-conda/

suggesting that one could create reproducible environments with conda. Might be an easy way to specify packages that would install on Linux, Windows, and Mac OSX by writing the appropriate yaml file.

**Comment by to266** [comment]
Actually most of us use conda - the problem, though, is that certain packages are (were?) not available for windows, namely `mayavi`. MKL-compiled numpy used to be lees than optimal as well, but rather recently that changed:

@ teoliphant at numpy/numpy
> I wanted to make sure and report back in this forum, the fact that Continuum is now giving away MKL Optimize licenses to all Anaconda users as I mentioned we would . It's not entirely obvious, but to receive the license, you need to get a free account on Anaconda Cloud (anaconda.org). The license will then be downloadable for you at this URL: https://anaconda.org/settings/add-ons. After you download the license, copy it to a directory named ~/.continuum on Linux and Mac or on Windows 7 and above: "C:\Users\<Your User Name>.continuum" (you might have to make that directory first).

While mayavi is occasionally a needed package, the main thing holding us back was the numpy. 

Then there is the ease of installation as well - most users don't want to fiddle with command-lines and terminal-looking things, especially in Windows environment - they expect a single *.exe to be run and in the end hyperspy should be ready to go. 

Finally, creating the yaml file would be trivially easy given our travis build, so we might as well add it

**Comment by jrminter** [comment]
I appreciate the willingness to create the yaml file. One of the major annoyances I find with python is "dependency hell" and having a yaml file for a release would make it easy to create/or restore a working environment for hyperspy. I did have lots of dependency problems with Anaconda 2.4.0 and went back to 2.3.0 for a base and so I have mayavi working on both Mac OSX (El Capitan)  and Windows x64 (I have stayed with Windows 7 - dragging my feet on upgrading to Windows 10).
